calico_containers/calico_ctl/endpoint.py

```diff
@@ -48,6 +49,39 @@
 from utils import print_paragraph
 
 
+def validate_arguments(arguments):
+    """
+    Validate argument values:
+        <PROFILES>
+
+    Arguments not validated:
+        <HOSTNAME>
+        <ORCHESTRATOR_ID>
+        <WORKLOAD_ID>
+        <ENDPOINT_ID>
+
+    :param arguments: Docopt processed arguments
+    """
+    # List of valid characters that Felix permits
+    valid_chars = '[a-zA-Z0-9_\.\-]'
+
+    # Validate Profiles
+    profile_ok = True
+    if "<PROFILES>" in arguments or "<PROFILE>" in arguments:
```

Comment:
I don't think this one has <PROFILES> as an option.

Also, is <PROFILE> a list or just a single profile name as it will change how it's handled.